src/components/Preferences/Language/index.tsx

```diff
@@ -50,23 +50,23 @@ const LanguagePreferences: React.FC<Props> = ({ isDetailed }) => {
           }}
           style={{ width: 120 }}
         >
-          <Option value="en">English</Option>
```

Comment:
let's preserve the ordering here, as much as I'd like the other languages to be used frequently, EN/FR/ES are my far the most used